src/components/veocean/LockOcean.svelte

```diff
@@ -133,7 +133,8 @@
           min={$lockedOceanAmount ? 0 : 1}
           max={parseFloat(getOceanBalance($connectedChainId)).toFixed(3)}
           error={$errors.amount}
-          disabled={getOceanBalance($connectedChainId) <= 0}
+          disabled={getOceanBalance($connectedChainId) <= 0 ||
+            new Date() > $oceanUnlockDate}
```

Comment:
My only concern is that once a lock ends, there will always be an old $oceanUnlockDate, and this will never reset properly.

I believe we should just use the same checks that the contract does when trying to create lock/extend lock. I.E. Wrap functionality from the contract into a helper function, and call that.

![Screenshot from 2022-09-15 12-45-25](https://user-images.githubusercontent.com/69865342/190495191-9bc763a7-8927-4c3e-8375-5896e113f678.png)